build.gradle

```diff
@@ -486,7 +486,7 @@ project(':iceberg-aws') {
     testImplementation 'software.amazon.awssdk:iam'
     testImplementation 'software.amazon.awssdk:s3control'
     testImplementation project(path: ':iceberg-api', configuration: 'testArtifacts')
-    testImplementation("com.adobe.testing:s3mock-junit4") {
+    testImplementation("com.adobe.testing:s3mock-junit5:2.17.0") {
```

Comment:
this should be defined in `versions.props` as `com.adobe.testing:s3mock-junit5 = 2.17.0` and then you don't need to specify the version number here

Reply:
also `com.adobe.testing:s3mock-junit4 = 2.11.0` can then be removed, since it's not used anymore